### PR TITLE
fix(react-theme): Update theme tokens

### LINKED ISSUE #25590: [Bug]: Token assignment change request
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
v9 https://aka.ms/fluentui-storybook
```

### Are you reporting Accessibility issue?

no

### Reproduction

React Components v9

### Bug Description

Requesting token assignment change for 2 shared colors (Plum and Dark Red)
In persona usecase, the background colors in dark theme were not equal in terms of brightness, resulting in these two looking more faded than other dark theme avatars. 

## Before
In dark theme slot, Plum Shade 30 was assigned
In dark theme slot, Dark Red Shade 30 was assigned

## After
In dark theme slot, assign Plum Shade 20
image

In dark theme slot, assign Dark Red Shade 20
image

image

Note: This only impact Plum and Dark Red, not the rest of the shared colors

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

_No response_

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #25600: [Bug]: New token addition requests (Alpha for skeleton component)
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
React Components v9
```

### Are you reporting Accessibility issue?

no

### Reproduction

https://aka.ms/fluentui-storybook

### 1. Adding Alpha Tokens for skeleton support over colored/images
image

cc @miroslavstastny (based on what we discussed this morning)

### 2. Modification on Background Overlay
Previously had Black Alpha 10 in the Light/Default theme slot, reassigning Black Alpha 40
image

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

Teams, any web product supporting native material on desktop offering

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #25603: [Bug]: Status color token assignments
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://aka.ms/fluentui-storybook
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://aka.ms/fluentui-storybook

### Change Request
Red Foreground 2 - Dark theme slot from Tint 20 to Tint 30
Red Border 2 - Dark theme slot from Tint 20 to Tint 30
Dark Orange Foreground 3 - Dark theme slot from Tint 20 to Tint 30
Dark Orange Border 2 - Dark theme slot from Tint 20 to Tint 30

### Final Expected Outcome:
image
image
image
image

Related to this bug, colors used inside Badge component: https://github.com/microsoft/fluentui/issues/22660
cc: @miroslavstastny , @redongreen , @behowell , @juraj

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### LINKED ISSUE #25721: [Bug]: Inverted token update request
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
https://aka.ms/fluentui-storybook
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://aka.ms/fluentui-storybook

## Request 1
**Actual Behavior**
image

**Expected Behavior**
Per update of the inverted background tokens in https://github.com/microsoft/fluentui/issues/24014, associated brand foreground inverted tokens need an update for dark theme as well.

In dark theme slots:
Brand Foreground Inverted: [Brand] 80
Brand Foreground Inverted Hover: [Brand] 70
Brand Foreground Inverted Pressed: [Brand] 60

edit on 12/5-- (Updated high contrast slot for Hover and Pressed) 
image

## Request 2

Red Foreground Inverted 
Columns: Light theme, Dark Theme, High contrast
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=34438%3A79739&t=F4ULpQUliliUcdAh-4

Green Foreground Inverted
Columns: Light theme, Dark Theme, High contrast
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=34438%3A79768&t=F4ULpQUliliUcdAh-4

Yellow Foreground Inverted
Columns: Light theme, Dark Theme, High contrast
image
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=34438%3A79713&t=F4ULpQUliliUcdAh-4

cc: @miroslavstastny, @redongreen 

### Logs

_No response_

### Requested priority

Normal

### Products/sites affected

https://aka.ms/fluentui-storybook

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by miroslavstastny**
Update `react-theme` tokens as requested by design.

## Added tokens
| Token | Light | Dark, TeamsDark | HC | Issue |
|---|---|---|---|---|
| `NeutralStencil1Alpha` |  Black Alpha 10 | White Alpha 10 | hcCanvasTex | #25600 |
| `NeutralStencil2Alpha` |  Black Alpha 5 | White Alpha 5 | hcCanvasText | #25600 |
| `RedForegroundInverted` | Red Tint 20 | Red Primary | hcCanvasText | #25721 |
| `GreenForegroundInverted` | Green Tint 20 | Green Primary | hcCanvasText | #25721 |
| `YellowForegroundInverted` | Yellow Tint 40 | Yellow Shade 30 | hcCanvasText | #25721 |

## Updated tokens
| Token | Theme | Before | After | Issue |
|---|---|---|---|---|
| `BackgroundOverlay` | Light | Black Alpha 10 | Black Alpha 40 | #25600 |
| `PlumBackground2` | Dark, TeamsDark | Plum Shade 30 | Plum Shade 20 | #25590 |
| `DarkRedBackground2` | Dark, TeamsDark | DarkRed Shade 30 | DarkRed Shade 20 | #25590 |
| `RedForeground3` | Dark, TeamsDark | Red Tint 20 | Red Tint 30 | #25603 |
| `RedBorder2` | Dark, TeamsDark | Red Tint 20 | Red Tint 30 | #25603 |
| `GreenForeground3` | Dark, TeamsDark | Green Tint 20 | Green Tint 40 | #25603 |
| `GreenBorder2` | Dark, TeamsDark | Green Tint 20 | Green Tint 40 | #25603 |
| `DarkOrangeForeground3` | Dark, TeamsDark | Dark Orange Tint 20 | Dark Orange Tint 30 | #25603 |
| `DarkOrangeBorder2` | Dark, TeamsDark | Dark Orange Tint 20 | Dark Orange Tint 30 | #25603 |
| `BrandForegroundInverted` | Dark, TeamsDark | Brand 100 | Brand 80 | #25721 |
| `BrandForegroundInvertedHover` | Dark, TeamsDark | Brand 110 | Brand 70 | #25721 |
| `BrandForegroundInvertedPressed` | Dark, TeamsDark | Brand 100 | Brand 60 | #25721 |
| `BrandForegroundInvertedHover` | HC | hcCanvasText | hcHighlightText | #25721 |
| `BrandForegroundInvertedPressed` | HC | hcCanvasText | hcHighlightText | #25721 |

## Related Issue(s)

* Fixes #25590
* Fixes #25600
* Fixes #25603
* Fixes #25721

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount_base.html">2850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount.html">2776</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Avatar-mount_base.html">1275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Avatar-mount.html">1266</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Button-mount_base.html">935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Button-mount.html">917</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProvider-mount_base.html">1501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProvider-mount.html">1486</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-mount_base.html">587</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-mount.html">593</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">544</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">543</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">583</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/MakeStyles-mount_base.html">1998</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/MakeStyles-mount.html">1988</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount_base.html">2850</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount.html">2776</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/Persona-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/SpinButton-mount_base.html">2368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25903/perf-test-react-components/SpinButton-mount.html">2327</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                      Change |
| :--------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `59.018 kB`<br />`16.371 kB` | `59.288 kB`<br />`16.435 kB` |     `270 B` <br />`64 B`  |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>               |  `33.48 kB`<br />`11.037 kB` |  `33.75 kB`<br />`11.101 kB` |     `270 B` <br />`64 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                     |  `29.722 kB`<br />`6.462 kB` |  `30.908 kB`<br />`6.714 kB` | `1.186 kB` <br />`252 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                   |   `17.51 kB`<br />`5.075 kB` |  `17.776 kB`<br />`5.141 kB` |     `266 B` <br />`66 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `79.294 kB`<br />`24.443 kB` |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `82.967 kB`<br />`21.022 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `51.076 kB`<br />`14.285 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |   `15.006 kB`<br />`6.009 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |  `67.306 kB`<br />`18.581 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `23.357 kB`<br />`7.361 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `24.247 kB`<br />`7.643 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `24.001 kB`<br />`7.049 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |    `32.923 kB`<br />`8.61 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `39.953 kB`<br />`9.933 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `37.611 kB`<br />`9.942 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `45.057 kB`<br />`11.323 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `48.753 kB`<br />`10.715 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `71.873 kB`<br />`20.936 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `67.651 kB`<br />`19.892 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `8.617 kB`<br />`3.62 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `9.792 kB`<br />`3.995 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `8.718 kB`<br />`3.674 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox (including child components)</abbr>                                                 |  `78.483 kB`<br />`25.136 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Dropdown.fixture.js'>Dropdown (including child components)</abbr>                                                 |  `77.712 kB`<br />`25.032 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `186.264 kB`<br />`52.303 kB` |
| <samp>react-dialog</samp> <br /> <abbr title='bundle-size/Dialog.fixture.js'>Dialog (including children components)</abbr>                                                    |  `83.036 kB`<br />`24.754 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `16.515 kB`<br />`5.921 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `10.836 kB`<br />`4.283 kB` |
| <samp>react-infobutton</samp> <br /> <abbr title='bundle-size/InfoButton.fixture.js'>InfoButton</abbr>                                                                        | `117.922 kB`<br />`35.391 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |    `24.194 kB`<br />`7.89 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |     `9.394 kB`<br />`3.88 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.862 kB`<br />`4.885 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `117.418 kB`<br />`36.256 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `120.487 kB`<br />`36.785 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                               |   `11.004 kB`<br />`4.188 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |   `57.113 kB`<br />`15.94 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 | `103.666 kB`<br />`31.858 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |   `10.495 kB`<br />`3.851 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `5.857 kB`<br />`1.978 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                                                                    |   `19.826 kB`<br />`7.417 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/Progress.fixture.js'>Progress</abbr>                                                                              |   `13.536 kB`<br />`5.107 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `15.817 kB`<br />`5.905 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/Radio.fixture.js'>Radio</abbr>                                                                                       |   `36.42 kB`<br />`12.126 kB` |
| <samp>react-radio</samp> <br /> <abbr title='bundle-size/RadioGroup.fixture.js'>RadioGroup</abbr>                                                                             |    `14.304 kB`<br />`5.72 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |   `23.474 kB`<br />`8.017 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |  `32.091 kB`<br />`10.183 kB` |
| <samp>react-spinbutton</samp> <br /> <abbr title='bundle-size/SpinButton.fixture.js'>SpinButton</abbr>                                                                        |  `44.502 kB`<br />`12.812 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |   `20.013 kB`<br />`6.446 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |  `33.427 kB`<br />`10.581 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `11.838 kB`<br />`4.625 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |    `15.148 kB`<br />`5.06 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `25.733 kB`<br />`8.328 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `42.032 kB`<br />`14.739 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f3e6e65b7f6435dc6b09bfd626e99921a10983d8'>f3e6e65b7f6435dc6b09bfd626e99921a10983d8</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts [hunk 1]
```diff
@@ -108,7 +108,9 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralBackgroundInvertedDisabled: whiteAlpha[10],
     colorNeutralStencil1: palette.neutralLight,
     colorNeutralStencil2: palette.neutralLighterAlt,
-    colorBackgroundOverlay: blackAlpha[10],
+    colorNeutralStencil1Alpha: inverted ? whiteAlpha[10] : blackAlpha[10],
+    colorNeutralStencil2Alpha: inverted ? whiteAlpha[10] : blackAlpha[5],
```
Comment (spmonahan) [inline apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts:112]: According to the PR description

```suggestion
    colorNeutralStencil2Alpha: inverted ? whiteAlpha[5] : blackAlpha[5],
```

File: packages/react-migration-v8-v9/src/components/Theme/v9ThemeShim.ts [hunk 2]
```diff
@@ -104,7 +104,9 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralBackgroundInvertedDisabled: whiteAlpha[10],
     colorNeutralStencil1: palette.neutralLight,
     colorNeutralStencil2: palette.neutralLighterAlt,
-    colorBackgroundOverlay: blackAlpha[10],
+    colorNeutralStencil1Alpha: inverted ? whiteAlpha[10] : blackAlpha[10],
+    colorNeutralStencil2Alpha: inverted ? whiteAlpha[10] : blackAlpha[5],
```
Comment (spmonahan) [inline packages/react-migration-v8-v9/src/components/Theme/v9ThemeShim.ts:108]: According to PR description

```suggestion
    colorNeutralStencil2Alpha: inverted ? whiteAlpha[5] : blackAlpha[5],
```

**Review by spmonahan — APPROVED** [review]

**Review by layershifter — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Pull request merged**